README.md

```diff
@@ -30,32 +30,48 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    .forEach calls the given function on each element of the object/array, .map will also do this for each element but will capture return values in a new array.
+
 2. What is the difference between a function and a method?
 
+    The main difference is that in the scope of an object, a function is reffered to as a 'method' of that object. A function on its own is not considered a 'method', just a standalone function, though it itself is an object.
+
 3. What is closure?
 
+    A closure is simply a function that has inherited a value from outside its own scope.
+
```

Comment:
A closure is a nested function that has access to it's outer functions variables. Closures store references to the outer functions variables even after function invocation has completed.